### PR TITLE
Add support for specifying validity hours when requesting a certificate

### PR DESCRIPTION
**Pull request by angelmoo**
Add the ability to specify validity when requesting a certificate
now use can request a certificate and specify validityHours: 

CertificateRequest certificateRequest = new CertificateRequest().subject(
		        new CertificateRequest.PKIXName()
		                .commonName("vcert-java.venafi.example")
		                .organization(Collections.singletonList("Example Company"))
		                .organizationalUnit(Arrays.asList("Example Division"))
		                .country(Collections.singletonList("US"))
		                .locality(Collections.singletonList("Salt Lake City"))
		                .province(Collections.singletonList("Utah")))
		        .dnsNames(Arrays.asList("alfa.venafi.example", "bravo.venafi.example", "charlie.venafi.example"))
		        .ipAddresses(Arrays.asList(InetAddress.getByName("10.20.30.40"),InetAddress.getByName("172.16.172.16")))
		        .emailAddresses(Arrays.asList("larry@venafi.example", "moe@venafi.example", "curly@venafi.example"))
		        .keyType(KeyType.RSA)
		        .validityHours(validityHours)
		        .issuerHint("MICROSOFT");

Added two tests cases to test the validity hours fix, one for TPP and one for cloud

All test ran well

### CODE REVIEW

File: src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+				break;
+			}
+
+			payload.caSpecificAttributes()
+			.add( new NameValuePair<String, String>(expirationDateAttribute, expirationDate) );
+		}
+
+	}
+
+	public static int getValidDays( int validHours ) {
```
Comment (marcos-albornoz) [inline src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java:69]: can you rename this method to getValidityDays()?

File: src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	public static void addExpirationDateAttribute( CertificateRequest request, CertificateRequestsPayload payload ) {
+
+		if ( request.validityHours() > 0 ) {
+
+			Instant now = Instant.now();
+			LocalDateTime utcTime = LocalDateTime.ofInstant(now, ZoneOffset.UTC);
+
+			int validityDays = request.validityHours() / 24;
+
+			if ( request.validityHours() % 24 > 0 ) {
```
Comment (marcos-albornoz) [inline src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java:23]: Can you put a little comment to explain why the validityDays is plus to 1 if the remainder of the division by 24 is more than 0?

File: src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java (hunk 1 above)
Comment (angelmoo) [reply src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java]: Fixed on the most recent commit

File: src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java (hunk 2 above)
Comment (angelmoo) [reply src/main/java/com/venafi/vcert/sdk/utils/VCertUtils.java]: fixed on the most recent commit

**Review by marcos-albornoz — APPROVED** [review]

**Review by marcos-albornoz — APPROVED** [review]

**Review by rvelaVenafi — APPROVED** [review]
code looks good

**Pull request merged**